packages/angular-test-app/src/preview-examples/toggle-ng-model.ts

```diff
@@ -10,9 +10,9 @@
 import { Component } from '@angular/core';
 
 @Component({
-  selector: 'app-example',
+  selector: 'app-example-ng-model',
```

Comment:
Same as previous comment. Will not comment additional occurrence.

Reply:
Same as above. Intentionally to keep this example from being used inside StackBlitz since it is an alternative.

Reply:
Okay but there is no need to add a special selector here. If its not referred in the documentation as playground it will never less not provided as stackblitz example